docs/devguide.md

```diff
@@ -277,8 +278,7 @@ helm install ../charts/catalog \
     --set apiserver.tls.ca=$(base64 --wrap 0 ${SC_SERVING_CA}) \
     --set apiserver.tls.cert=$(base64 --wrap 0 ${SC_SERVING_CERT}) \
     --set apiserver.tls.key=$(base64 --wrap 0 ${SC_SERVING_KEY}) \
-    --set apiserver.image=quay.io/kubernetes-service-catalog/apiserver:canary \
-    --set controllerManager.image=quay.io/kubernetes-service-catalog/controller-manager:canary
```

Comment:
I think we still might want separate values to be able to be set here.

Reply:
See above where I argue against this.